client/src/app/modules/module-instance/components/fields/chips/chips.component.ts

```diff
@@ -0,0 +1,48 @@
+import {ChangeDetectionStrategy, Component, OnInit} from '@angular/core';
+import {MatChipInputEvent} from '@angular/material';
+import {FieldComponent, FieldData} from '../../field/field.component';
+
+interface ChipsData extends FieldData {
+  selectable: boolean;
+  removable: boolean;
+  addOnBlur: boolean;
+}
+
+@Component({
+  selector: 'jms-chips',
+  templateUrl: './chips.component.html',
+  styleUrls: ['./chips.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ChipsComponent extends FieldComponent<ChipsData>
+  implements OnInit {
+  data = this.cData.control.value;
+
+  ngOnInit() {
```

Comment:
Let's remove this, I need to adjust the control to have the proper value so this won't be needed.